src/Microsoft.Identity.Web/ClaimConstants.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.Identity.Web
     /// <summary>
     /// Constants for claim types.
     /// </summary>
-    internal static class ClaimConstants
+    public static class ClaimConstants
```

Comment:
Is there a class in the .NET/Core/ASP.NET that we can just use for these constants? Wondering since if we expose this, we allow users to take dependency on this and maybe we can reuse something existing. But if it's a small enough class that will stay constant, maybe it's fine.